app/views/hyrax/admin/admin_sets/_form_participants.html.erb

```diff
@@ -1,10 +1,10 @@
-<%# OVERRIDE Hyrax v3.4.2 Use Hyrax::Groups for groups select box %>
+<%# OVERRIDE Hyrax v5.0.0rc2 Use Hyrax::Groups for groups select box %>
 <div id="participants" class="tab-pane">
-  <div class="panel panel-default labels">
-    <div class="panel-body">
-      <h2><%= t('.add_participants') %></h2>
+  <div class="card labels edit-sharing-tab">
+    <div class="card-body">
+      <h2 class="h3"><%= t('.add_participants') %></h2>
       <% access_options = options_for_select([['Manager', 'manage'], ['Depositor', 'deposit'], ['Viewer', 'view']]) %>
-      <%= simple_form_for @form.permission_template,
+      <%= simple_form_for collection_permission_template_form_for(form: @form),
```

Comment:
Wondering about this.  My read, without looking at `#collection_permission_template_form_for` is that `collection_permission_template_form_for` would be similar to `simple_form_for`.  Meaning that we're basically doing `simple_form_for(simple_form_for(form: @form))`; I think this may need further investigation.

Reply:
Hmmm, yeah I can see that.  I didn't do a deep dive into that method and just deferred to Hyrax's implementation
https://github.com/samvera/hyrax/blob/double_combo/app/views/hyrax/admin/admin_sets/_form_participants.html.erb#L8